components/routing/composer.json

```diff
@@ -1,11 +1,11 @@
 {
     "require": {
-        "illuminate/routing": "^5.5",
-        "illuminate/events": "^5.5"
+        "illuminate/routing": "5.5.*",
+        "illuminate/events": "5.5.*"
     },
     "autoload": {
         "psr-4": {
-            "App\\": "src/"
+            "Torch\\Routing\\": "src/"
```

Comment:
I prefer `App`

Reply:
I made a note in the commit description about why I did this. PhpStorm warns about duplicate class definitions if more than one component has the same FQCN. It doesn't functionally impact the code, but that was why.

Reply:
Thanks--I'd still prefer `App`, and storm can just deal with it. Thanks!